src/style.css

```
@@ -1 +1,38 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  padding: 0;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  align-items: center;
+  gap: 4px;
+}
+
+.stars--0 :nth-child(n + 1),
+.stars--1 :nth-child(-n + 5),
+.stars--2 :nth-child(-n + 5),
+.stars--3 :nth-child(-n + 5),
+.stars--4 :last-child {
+  background-image: url(./images/star.svg);
+  color: #f3f3f3;
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(n + 1) {
+  background-image: url(./images/star-active.svg);
+  color: #ffde6a;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

```suggestion
.stars__star {
  width: 16px;
  height: 16px;
  background-repeat: no-repeat;
  background-position: center;
  background-image: url(./images/star.svg);
}
```
just add image into .stars__star